analyzers/its/expected/akka.net/Akka.Streams--netstandard2.0-S3218.json

```diff
@@ -0,0 +1,17 @@
+{
+"issues":  [
+{
+"id":  "S3218",
+"message":  "Rename this field to not shadow the outer class' member with the same name.",
+"location":  {
+"uri":  "sources\akka.net\src\core\Akka.Streams\Implementation\ActorMaterializerImpl.cs",
```

Comment:
True positive. There is a static `Props` method in the class that contains the class that contains the marked property.

```cs
    public class StreamSupervisor : ActorBase
    {
        public sealed class Materialize : INoSerializationVerificationNeeded, IDeadLetterSuppression
        {
            public readonly Props Props; // Line 537
        }
        public static Props Props(ActorMaterializerSettings settings, AtomicBoolean haveShutdown) // Line 626
            => Actor.Props.Create(() => new StreamSupervisor(settings, haveShutdown)).WithDeploy(Deploy.Local);
    }
```